config/locales/client.en.yml

```
@@ -4180,6 +4183,9 @@ en:
         this_week: "Week"
         today: "Today"
 
+    custom_homepage:
+      admin_message: 'Your themes has enabled the "custom_homepage" modifier but it does not output anything in the [custom-homepage] connector. (This message is only shown to site administrators.)'
```

```suggestion
      admin_message: 'One of your themes has enabled the "custom_homepage" modifier but it does not output anything in the [custom-homepage] connector. (This message is only shown to site administrators.)'
```

(or 'your themes have' instead of 'your themes has')

---

Yeah, of course, thanks!